projects/plugins/jetpack/class-jetpack-xmlrpc-methods.php

```diff
@@ -30,10 +30,12 @@ public static function init() {
 	 */
 	public static function xmlrpc_methods( $methods ) {
 
-		$methods['jetpack.featuresAvailable'] = array( __CLASS__, 'features_available' );
-		$methods['jetpack.featuresEnabled']   = array( __CLASS__, 'features_enabled' );
-		$methods['jetpack.disconnectBlog']    = array( __CLASS__, 'disconnect_blog' );
-		$methods['jetpack.jsonAPI']           = array( __CLASS__, 'json_api' );
+		$methods['jetpack.featuresAvailable']           = array( __CLASS__, 'features_available' );
+		$methods['jetpack.featuresEnabled']             = array( __CLASS__, 'features_enabled' );
+		$methods['jetpack.disconnectBlog']              = array( __CLASS__, 'disconnect_blog' );
+		$methods['jetpack.getUserlessPurchaseToken']    = array( __CLASS__, 'get_userless_purchase_token' );
```

Comment:
I believe we're trying to move away from the "userless" naming now. See #19734. Maybe we should avoid introducing such changes here now? cc @leogermani

Reply:
Yes! Please refer to p1HpG7-bRQ-p2